packages/trie-hash/test/data.ts

```diff
@@ -0,0 +1,263 @@
+// Copyright 2017-2018 @polkadot/trie-hash authors & contributors
```

Comment:
These are basically the Rust tests - as a packaged file so we can use it both in trie-hash and trie-db to check the results